components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -1430,15 +1461,22 @@ public String exportSPApplicationFromAppID(String applicationId, boolean exportS
     public String exportSPApplication(String applicationName, boolean exportSecrets, String tenantDomain)
             throws IdentityApplicationManagementException {
 
+        ServiceProvider serviceProvider = exportSP(applicationName, exportSecrets, tenantDomain);
+        return marshalSP(serviceProvider, tenantDomain);
+    }
+
+    public ServiceProvider exportSP(String applicationName, boolean exportSecrets, String tenantDomain)
```

Comment:
For each and every public methods better to have a method level comments.

Reply:
Added method level comments for the public methods.